staging/traefik/README.md

```diff
@@ -216,6 +216,7 @@ The following table lists the configurable parameters of the Traefik chart and t
 | `metrics.prometheus.buckets`           | A list of response times (in seconds) - for each list element, Traefik will report all response times less than the element. | `[0.1,0.3,1.2,5]`                                 |
 | `metrics.prometheus.service.name`      | Custom service name for the Prometheus service                                                 | ``                                           |
 | `metrics.prometheus.service.annotations`      | Annotations for the Prometheus service                                                 | `{ prometheus.io/scrape: "true" }`                                           |
+| `metrics.prometheus.service.labels`    | Labels for the Prometheus service                                               | `{}`                                           |
```

Comment:
Just curious: why we introduced label instead of `annotations`? Most of the prometheus setting goes into `annotations`

Reply:
Im curious too